services/policies/pkg/engine/opa/engine.go

```diff
@@ -57,5 +57,7 @@ func (o OPA) Evaluate(ctx context.Context, qs string, env engine.Environment) (b
 		return false, err
 	}
 
-	return result.Allowed(), nil
+	allowed := result.Allowed()
+
+	return allowed, nil
```

Comment:
```suggestion
	return result.Allowed(), nil
```
`allowed` var not needed

Reply:
yup, had it like that before, decided to have it as var to set breakpoints on return and evaluate the result more easy... i can revert it if you prefer